### ISSUE TITLE
[PyTorch/Transformer] WMT data set versions

### ISSUE THREAD

**Issue by gieflij** [NVIDIA/DeepLearningExamples#875]
Hello, now I am using Transformer for inference.
I have a question about WMT data set versions.

I followed **quick start guide except training (fourth step)**, finished inference and tested model checkpoint accuracy.
I downloaded Transformer checkpoints from NVIDIA NGC. The checkpoint's model is vaswani-wmt-en-de-big model and it was trained with WMT14 en-de dataset.

By the way, I want to test other versions of WMT e.g., WMT13.
I can't train the model with other versions of WMT. 
Thus, I only can use WMT14 checkpoints and bpe-codes for WMT14.
Is it okay to use other versions of WMT when I test the model's accuracy?
For example, the code is
**sacrebleu -t wmt13 -l en-de --echo src | python inference.py --buffer-size 5000 --path /checkpoints/transformer_pyt_20.06.pt --max-tokens 10240 --fuse-dropout-add --remove-bpe --bpe-codes /data/wmt14_en_de_joined_dict | sacrebleu -t wmt13 -l en-de -lc**

- transformer checkpoints: transformer_pyt_20.06.pt (vaswani_big_model, trained with wmt14_en_de data set)
- bpe-codes: wmt14_en_de_joined_dict

Thank you

**Comment by jbaczek** [comment]
What do you mean by that you can't train the model with other versions of WMT? Does it not converge?
You can test your model against another version's test set. It is a perfectly valid thing to do. But why do you want to do this?
You can find full list of supported test sets in the sacrebleu repository. 
https://github.com/mjpost/sacrebleu/blob/5dfcaa3cee00039bcad7a12147b6d6976cb46f42/sacrebleu/dataset.py#L24

**Comment by gieflij** [comment]
I mean, I only have one GPU so I need to spend too much times for train and getting checkpoints.
I already downloaded checkpoints from NVIDIA. It was trained with WMT14 en-de data set.
I can get sacrebleu score for other versions of WMT, e.g., WMT13, with WMT14 checkpoints.
Is it okay to get sacrebleu with different versions of checkpoints and data set?
Or Is it necessary to use same versions of them?

**Comment by gieflij** [comment]
> What do you mean by that you can't train the model with other versions of WMT? Does it not converge?
> You can test your model against another version's test set. It is a perfectly valid thing to do. But why do you want to do this?
> You can find full list of supported test sets in the sacrebleu repository.
> https://github.com/mjpost/sacrebleu/blob/5dfcaa3cee00039bcad7a12147b6d6976cb46f42/sacrebleu/dataset.py#L24

I mean, I only have one GPU so I need to spend too much times for train and getting checkpoints.
I already downloaded checkpoints from NVIDIA. It was trained with WMT14 en-de data set.
I can get sacrebleu score for other versions of WMT, e.g., WMT13, with WMT14 checkpoints.
Is it okay to get sacrebleu with different versions of checkpoints and data set?
Or Is it necessary to use same versions of them?

**Comment by jbaczek** [comment]
Our model was trained and validated on WMT14 thus the numbers in repo refer to that. Of course, you can evaluate our model on WMT13 but it has different test set and the results will differ
